app/src/main/java/campus/tech/kakao/map/MapRepository.kt

```diff
@@ -0,0 +1,38 @@
+package campus.tech.kakao.map
+
+import android.content.Context
+import android.database.sqlite.SQLiteDatabase
+
+class MapRepository(context: Context) {
+    private var localDB: PlacesDBHelper
+
+    init {
+        localDB = PlacesDBHelper(context)
+        val dbFile = context.getDatabasePath("PlaceTable.db")
+        if (!dbFile.exists())
+            insertInitialData()
```

Comment:
if문을 여러줄에 걸쳐서 작성하실 때에는 중괄호를 붙여주시는 것이 좋습니다.
```suggestion
        if (!dbFile.exists()) {
            insertInitialData()
        }
```
🔗 코틀린 스타일 가이드 > [중괄호](https://developer.android.com/kotlin/style-guide?hl=ko#braces)